packaging/rpm/microshift.spec

```diff
@@ -157,6 +157,7 @@ restorecon -v %{buildroot}%{_bindir}/microshift
 
 install -d -m755 %{buildroot}%{_sysconfdir}/crio/crio.conf.d
 install -p -m644 packaging/crio.conf.d/microshift.conf %{buildroot}%{_sysconfdir}/crio/crio.conf.d/microshift.conf
+install -p -m644 packaging/crio.conf.d/microshift-ovn.conf %{buildroot}%{_sysconfdir}/crio/crio.conf.d/microshift-ovn.conf
```

Comment:
In the case there are two microshift*.conf installed, which one takes preference?

Reply:
I think CRI-O can take several config files. It's just a way to arrange configs.

Reply:
Correct, alphabetical order, all options "added"